lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -11,15 +11,25 @@ def self.email(params = {}, _secrets = {}, _context = {})
         end
 
         private_class_method def self.email_status!(runner_context)
+          miq_task_status!(runner_context)
+        end
+
+        private_class_method def self.miq_task_status!(runner_context)
           miq_task = ::MiqTask.find(runner_context["miq_task_id"])
```

Comment:
I think `find` throws an exception if the record is not found

I have this change locally:

```suggestion
          miq_task = ::MiqTask.find_by(:id => runner_context["miq_task_id"])
```